framework/global/ret.h

```diff
@@ -33,7 +33,7 @@ class Ret
         UnknownError    = 1,
 
         // not error, just codes
-        Cancel       = 3,  // abort by user
```

Comment:
I think it would be useful to remove similar art indentation so as not to do it every single time this enum extends